docs/docs/en/guide/datasource/athena.md

```diff
@@ -0,0 +1,21 @@
+# AWS Athena
+
+![mysql](../../../../img/new_ui/dev/datasource/athena.png)
+
+## Datasource Parameters
+
+| **Datasource** | **Description** |
+| --- | --- |
+| Datasource | Select ATHENA. |
+| Datasource name | Enter the name of the DataSource. |
+| Description | Enter a description of the DataSource. |
+| Username | Set the AWS access key. |
+| Password | Set the AWS secret access key. |
+| AwsRegion | Set the AWS region. |
+| Database name | Enter the database name of the ATHENA connection. |
+| Jdbc connection parameters | Parameter settings for ATHENA connection, in JSON format. |
+
+## Native Supported
+
+No, read section example in [datasource-setting](../howto/datasource-setting.md) `DataSource Center` section to activate this datasource.
```

Comment:
If not native supported, can we provider the JDBC connection name to download, better add some download link for it